src/services/server/station.service.ts

```
@@ -0,0 +1,275 @@
+    const scored = filtered.map((result) => ({
+      ...result,
+      score: this.calculateStationScore(result, location),
+    }));
+
+    // Return highest scoring stations
+    return scored.sort((a, b) => b.score - a.score).slice(0, count);
+  }
+
+  calculateStationScore(
+    result: NearestStationResult,
+    location: Coordinates
```

ESLint

[error] 'location' is defined but never used.

(@typescript-eslint/no-unused-vars)